commands/cluster-shards.md

```diff
@@ -32,8 +35,8 @@ The current list of attributes:
 * endpoint: The preferred endpoint to reach the node, see below for more information about the possible values of this field.
 * ip: The IP address to send requests to for this node.
 * hostname (optional): The announced hostname to send requests to for this node.
-* port (optional): The TCP port number of the node.
-* tls-port (optional): The TLS port of the node.
+* port (optional): The TCP (non-TLS) port of the node. At least one of port or tls-port will be present.
```

Comment:
not related to this PR but I think that we should add some tests to validate the contract defined here.